breadability/document.py

```diff
@@ -4,30 +4,15 @@
 
 from __future__ import absolute_import
 
-import re
 import logging
-import chardet
-
-from lxml.etree import (
-    tounicode,
-    XMLSyntaxError,
-)
-from lxml.html import (
-    document_fromstring,
-    HTMLParser,
-)
+import re
 
-from ._compat import (
-    to_bytes,
-    to_unicode,
-    unicode,
-    unicode_compatible,
-)
-from .utils import (
-    cached_property,
-    ignored,
-)
+import chardet
+from lxml.etree import ParserError, XMLSyntaxError, tounicode
+from lxml.html import HTMLParser, document_fromstring
```

Comment:
This is a style-thing but I prefer the import ( foo, bar ) with newlines over packing everything on one line. It also helps with checking to see which packages are added / removed.